assets/xml/objects/object_fr.xml

```diff
@@ -1,67 +1,68 @@
+        <Texture Name="gFrogLowerInnerMouthTex" OutName="frog_lower_inner_mouth" Format="rgba16" Width="32" Height="32" Offset="0x3BA0" />
+        <Texture Name="gFrogUpperInnerMouthTex" OutName="frog_upper_inner_mouth" Format="rgba16" Width="32" Height="32" Offset="0x43A0" />
+        <Texture Name="gFrogSkinTex" OutName="frog_skin" Format="rgba16" Width="16" Height="16" Offset="0x4BA0" />
+        <Texture Name="gFrogUpperBodyPatternTex" OutName="frog_upper_body_pattern" Format="rgba16" Width="32" Height="32" Offset="0x4DA0" />
+        <Texture Name="gFrogLowerBodyPatternTex" OutName="frog_lower_body_pattern" Format="i8" Width="32" Height="32" Offset="0x55A0" />
+        <Texture Name="gFrogIrisOpenTex" OutName="frog_iris_open" Format="rgba16" Width="16" Height="16" Offset="0x59A0" />
+        <Texture Name="gFrogIrisClosedTex" OutName="frog_iris_closed" Format="rgba16" Width="16" Height="16" Offset="0x5BA0" />
+        <Texture Name="gFrogEyeTex" OutName="frog_eye" Format="ia16" Width="32" Height="32" Offset="0x5DA0" />
+        <Texture Name="gFrogHandAndToeTex" OutName="frog_hand_and_toe" Format="ia16" Width="32" Height="32" Offset="0x65A0" />
+        <DList Name="gFrogRightEyeDL" Offset="0x8EC0" />
+        <DList Name="gFrogLeftEyeDL" Offset="0x8F60" />
+        <DList Name="gFrogHeadDL" Offset="0x9000" />
+        <DList Name="gFrogRightIrisDL" Offset="0x9328" />
+        <DList Name="gFrogLeftIrisDL" Offset="0x93C8" />
+        <DList Name="gFrogJawDL" Offset="0x9468" />
+        <DList Name="gFrogUpperBodyDL" Offset="0x95A0" />
+        <DList Name="gFrogLeftForearmDL" Offset="0x98B8" />
+        <DList Name="gFrogLeftHandDL" Offset="0x9AE0" />
+        <DList Name="gFrogLeftUpperArmDL" Offset="0x9B80" />
+        <DList Name="gFrogRightForearmDL" Offset="0x9D88" />
+        <DList Name="gFrogRightHandDL" Offset="0x9FB0" />
+        <DList Name="gFrogRightUpperArmDL" Offset="0xA050" />
+        <DList Name="gFrogLowerBodyDL" Offset="0xA258" />
+        <DList Name="gFrogRightHeelDL" Offset="0xA428" />
+        <DList Name="gFrogRightToeDL" Offset="0xA6B8" />
+        <DList Name="gFrogRightShinDL" Offset="0xA758" />
+        <DList Name="gFrogRightThighDL" Offset="0xA978" />
+        <DList Name="gFrogLeftHeelDL" Offset="0xABA8" />
+        <DList Name="gFrogLeftToeDL" Offset="0xAE38" />
+        <DList Name="gFrogLeftShinDL" Offset="0xAED8" />
+        <DList Name="gFrogLeftThighDL" Offset="0xB0F8" />
+        <DList Name="gFrogDustMaterialDL" Offset="0xB328" />
+        <DList Name="gFrogDustModelDL" Offset="0xB338" />
+        <Limb Name="gFrogLowerBodyLimb" Type="Standard" EnumName="FROG_LIMB_LOWER_BODY" Offset="0xB3C8" />
+        <Limb Name="gFrogUpperBodyLimb" Type="Standard" EnumName="FROG_LIMB_UPPER_BODY" Offset="0xB3D4" />
+        <Limb Name="gFrogRootLimb" Type="Standard" EnumName="FROG_LIMB_ROOT" Offset="0xB3E0" />
```

Comment:
I would probably call this `gFrogHeadRootLimb`. The "head" part is because all the head DLs (the head itself, the eyes/irises, the jaw) are children of this limb, but no other limbs are. Thus, it's the "root" of all head pieces.